ntbs-service/Services/NotificationService.cs

```diff
@@ -164,11 +179,30 @@ public async Task SubmitNotification(Notification notification)
             notification.NotificationStatus = NotificationStatus.Notified;
             notification.SubmissionDate = DateTime.UtcNow;
 
-            await context.SaveChangesAsync();
+            await UpdateDatabase(isSubmission: true);
         }
         
         public async Task<Notification> GetNotificationWithAllInfoAsync(int? id) {
             return await repository.GetNotificationWithAllInfoAsync(id);
         }
+
+        private async Task UpdateDatabase(bool? isSubmission = null) {
```

Comment:
Why don't we pass in the audit type directly? we can default to `AuditType.Edit` and get the same behaviour, but it'll be more straight-forward and easier to extend?

Reply:
Yes good point :)